### ISSUE TITLE
`Invalid type for parameter RoleArn` when setting `"manage_roles": false`

### ISSUE THREAD

**Issue by smoll** [Miserlou/Zappa#276]
First off, thanks for the amazing project.

I've been deploying from my local development environment (`zappa update dev`) just fine with my `zappa_settings.json` containing

```json
{
    "dev": {
        "app_function": "myapp.app",
        "s3_bucket": "dev-nameofmybucket"
    }
}
```

However, I was starting to look at deploying from my CI server, and over the course of investigating that, I discovered that changing my settings file to

```json
{
    "dev": {
        "app_function": "myapp.app",
        "manage_roles": false,
        "role_name": "ZappaLambdaExecution",
        "s3_bucket": "dev-nameofmybucket"
    }
}
```

breaks things. Here's some sample output:

```
Packaging project as zip...
Uploading zip (9.6MiB)...
100%|████████████████████████████████████████████████████████████████████████████| 10.1M/10.1M [00:00<00:00, 14.5MB/s]
Updating Lambda function code..
Scheduling..
Oh no! An error occured! :(

==============

Traceback (most recent call last):
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/src/zappa/zappa/cli.py", line 1119, in handle
    sys.exit(cli.handle())
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/src/zappa/zappa/cli.py", line 176, in handle
    self.update()
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/src/zappa/zappa/cli.py", line 341, in update
    self.schedule()
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/src/zappa/zappa/cli.py", line 479, in schedule
    events=events
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/src/zappa/zappa/zappa.py", line 1158, in schedule_events
    RoleArn=self.credentials_arn
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/lib/python2.7/site-packages/botocore/client.py", line 159, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/lib/python2.7/site-packages/botocore/client.py", line 470, in _make_api_call
    api_params, operation_model, context=request_context)
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/lib/python2.7/site-packages/botocore/client.py", line 523, in _convert_to_request_dict
    api_params, operation_model)
  File "/Users/smoll/workspace/poc-zappa/flask-poc/env/lib/python2.7/site-packages/botocore/validate.py", line 270, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
ParamValidationError: Parameter validation failed:
Invalid type for parameter RoleArn, value: ('arn:aws:iam::123456789012:role/ZappaLambdaExecution', False), type: <type 'tuple'>, valid types: <type 'basestring'>

==============
```

I tried updating to the latest version of `zappa` on github (replace `zappa==0.21.0` in my requirements.txt with `-e git+https://github.com/Miserlou/Zappa@master#egg=zappa`) and it doesn't help things.

Let me know if there's anything I can do to help gather more information about this.

**Comment by Miserlou** [comment]
Excellent bug report, thank you. Will investigate.

**Comment by Miserlou** [comment]
Ah, I think I see what this is.

**Comment by Miserlou** [comment]
Should be fixed now, thanks for reporting!

**Comment by smoll** [comment]
Thanks for the fast fix, works beautifully now!

Just a quick correction to something I posted above. To use the latest code on master installed via a requirements.txt file, instead of using

```
-e git+https://github.com/Miserlou/Zappa@master#egg=zappa
```

we should be using

```
git+https://github.com/Miserlou/Zappa@master#egg=zappa
```

When I used the former, and I deployed, I saw a lot of `[1471990270125] module initialization error: Attempted relative import in non-package` in the `zappa tail dev` logs. After fixing the installation & redeploying, I was back in business.
